internal/tests/pytests/requirements.txt

```diff
@@ -0,0 +1,19 @@
+certifi==2023.7.22
```

Comment:
Why not a `requirements.in` that is compiled with `pip-compile` to check on versioning? Specifying the versions in `requirements.txt` without this (whether it be `poetry`/`pipenv`/`choose-your poison` is prone to error. Using `pip-compile` (and others) ensures you have _all_ requirements, not just the ones you think you need. 

Additionally, why is this dev req hidden so deep in the directory structure? It's dev, but still top-level requirements files are easier to manage/build for your project. Would be beneficial to place it up top with your prod `requirements.txt` and rename to `requirements-dev.txt` then you can _include_ `requirements.txt` in your `requirements-dev.txt` (if using `pip-compile` include `requirements.in` in your `requirements-dev.in`) and `pip` will know what to do.

Reply:
I created GitHub issue #2289 to track this enhancement.